src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -6,6 +6,7 @@
     <Title>Datadog APM</Title>
     <Description>Manual instrumentation library for Datadog APM</Description>
     <Authors>lucas.pimentel.datadog;colinhigginsdatadog;zachmontoyadd</Authors>
+    <RunAnalyzersDuringBuild>true</RunAnalyzersDuringBuild>
```

Comment:
Do we need this? This property is already `true` by default.
```suggestion
```

Reply:
I had manually turned it off to keep StyleCop from cramping my style :) I'll remove it.

Reply:
> to keep StyleCop from cramping my style

Ah, yes. I know the feeling. If having this makes it easier to toggle StyleCop temporarily while working, then I think it's fine to leave it, although we might want to move it into one of the shared `Directory.Build.props` files.

Reply:
thanks -- I'll look into that too.